pyClient/commands/zeth

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
```

Comment:
Add copyright header please

Reply:
Added copyright header (and updated script to catch these)